File: programme/subtract-two-numbers/subtract-two-numbers.cs
// Write a programme to subtract two numbers
using System;

public class Subtraction
{
    static void Main(string[] args)
    {
        int Subtract(int a, int b)
        {
            return a - b;
        }

        // Input a
        Console.Write("Enter a: ");
        int a = int.Parse(Console.ReadLine());
        // Input b
        Console.Write("Enter b: ");
        int b = int.Parse(Console.ReadLine()); ;

        // Difference of numbers 
        Console.WriteLine("Difference of numbers: " + Subtract(a, b));
    }
}

File: programme/subtract-two-numbers/subtract-two-numbers.js
// Write a programme to subtract two numbers

"use strict";

// perform following command to run programme
// node subtract-two-numbers.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Input  : ", (nums) => {
  let [x, y] = nums.split(" ").map(Number);
  if (isNaN(x) || isNaN(y)) {
    console.error("Some error occured");
  } else {
    console.log(`Òutput : ${x - y}`);
  }
  rl.close();
});
